### ISSUE TITLE
[Screen Reader-Social Embedded-Social Activities]: TalkBack does not read the full forms of the initials used for time (weeks, hours, seconds etc.)

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#25]
Priority: High

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Create any profile and comment on that profile post.
8. Navigate to ""reply"" button and reply on that comment.
9. Navigate the page  via swipe gesture.
10.Verify the speech of Talk Back for initials used for time (weeks, hours, seconds etc.)

Expected
TalkBack should read the full forms of the initials used for time (weeks, hours, minute, seconds etc.), Talk Back should announce  ""25 week"" , ""9 hour"", ""0 seconds"" etc.

Actual
TalkBack does not read the full forms of the initials used for time (weeks, hours, minute, seconds etc.), Talk Back is annoucing ""25w"" , ""9h"" etc. which is quite confusing for screen reader users to understand .

MAS Rule: MAS1.3.1

Impact
User who is rely on screen reader will get confuse d and face difficulty if  TalkBack does not read the full forms of the initials used for time (weeks, hours, minute, seconds etc.).

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
